.changeset/bright-dryers-attend.md

```diff
@@ -0,0 +1,6 @@
+---
+'astro': patch
+'@astrojs/lit': patch
+---
+
+Implement client:only functionality in Lit and add lit to the client:only warning
```

Comment:
Is this a new "feature" (did client:only not work at all in this integration before?)

Reply:
It did not before. Was wondering if you'd consider this a bugfix or feature. If minor, do you know if I can update that inline or do I have to generate a new changeset?

Reply:
So I was thinking, `@astrojs/lit`: minor and `astro`: patch because the only thing changed in astro core was a warning update?

Reply:
Did the above resolving convo, pls reopen if you think it should change